lib/eparser.js

```diff
@@ -113,9 +114,45 @@ function getFieldsSync(filename, prefix) {
     });
   })
 
+  contract.is.map(is => {
+    const importedFileName = is.name;
+    const from = graph.body.filter(member => {
+      if(!member.from) return false;
+      return member.from.endsWith(`${importedFileName}.sol`); // assuming only solidity files will be imported
+    })[0].from;
+
+    return buildImportFilePath(from.split('/'), filename.split('/')).join('/'); // assuming only unix like directory separation
```

Comment:
We should use the path module in nodejs for this. That will ensure its not paltform dependant.